jfix-stdlib-ratelimiter/src/main/kotlin/ru/fix/stdlib/ratelimiter/RateLimiterKt.kt

```diff
@@ -0,0 +1,31 @@
+package ru.fix.stdlib.ratelimiter
+
+import java.time.temporal.TemporalUnit
+
+interface RateLimiterKt : AutoCloseable {
+
+    /**
+     * Get configured rate limit
+     */
+    suspend fun getRate(): Double
```

Comment:
Are you sure that method for getting currently configured rate limit must be suspenable?

Reply:
Removed suspend